skorch/callbacks/logging.py

```diff
@@ -875,3 +875,70 @@ def on_epoch_end(self, net, **kwargs):
 
         for key in filter_log_keys(epoch_logs.keys(), self.keys_ignored_):
             self.experiment.log_scalar(key + self.epoch_suffix_, epoch_logs[key], epoch)
+
+
+class MLflow(Callback):
+    """
+    >>> import mlflow
+    >>> net = NeuralNetClassifier(net, callbacks=[MLflow])
+    >>> with mlflow.start_run():
+    ...     net.fit(X, y)
+    """
+    def __init__(
+        self,
+        log_on_batch_end=False,
+        log_on_epoch_end=True,
+        keys_ignored=[],
+        create_artifact=True,
+        logged_parameters=["lr", "batch_size"],
+    ):
+        import mlflow
+        self.mlflow = mlflow
```

Comment:
Is this the correct way to get the library without raising an exception if the user doesn't have a use for this callback?

Reply:
See my [comment](https://github.com/skorch-dev/skorch/pull/770#issuecomment-833780968)